src/map/map.ts

```diff
@@ -774,6 +774,16 @@ export class PropertiesMap {
         return this._selectTrace<number[]>(values, selected, trace);
     }
 
+    /** Prepare a title of an axis with potential units */
+    private _title(name: string): string{
+        let title = name;
+        let units = this._property(title).units;
+            if (units !== undefined){
+                title += ` [${units}]`;
+            }
+        return title;
+    }
+    
```

Comment:
```suggestion
    /** Prepare a title of an axis with potential units */
    private _title(name: string): string{
        if(this._property(name).units !== undefined) {
            return name + ` [${units}]`;
        } else { return name; }
    }
    
```

Just to make it more compact

Reply:
Single line else are not part of our prettier config ^_^

But why not something like

```ts
private _title(name: string): string{
    const units = this._property(name).units;
    if (units !== undefined) {
        return name + ` [${units}]`;
    } else {
        return name;
    }
}
```

Reply:
Yah, that would probably work best... 

Side note: `prettier` is wrong, single line `else`s are awesome. But alas.

Reply:
> Side note: prettier is wrong, single line elses are awesome. But alas.

Only if you are trying to win [IOCCC](https://www.ioccc.org/) and write code like [this](https://www.ioccc.org/2019/adamovsky/prog.c) 😁

Reply:
```suggestion
    private _title(name: string): string{
        if (name !== '') {
            const units = this._property(name).units;
            if (units !== undefined) {
                return name + ` [${units}]`;
            } 
        }
        return name
    }
    
```

What about adding the catchall so that we can use `this._title` for the `zaxis` for consistency?